### ISSUE TITLE
abnormal circulating orosomucoid

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3096]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a. abnormal circulating orosomucoid 1
b. increased circulating orosomucoid 1
c.  decreased circulating orosomucoid 1

>2. Synonyms (e.g., absent spleen)

For orosomucoid 1: alpha1-acid-glycoprotein, acute-phase reactant AGP, acute-phase protein alpha1-acid-glycoprotein

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a. deviation from the normal levels of orosomucoid 1 found in the blood shortly after infection and which participates in host defense against infection
b.  increased levels of orosomucoid 1, which participates in the acute-phase reaction
c.  decreased levels of orosomucoid 1, which participates in the acute-phase reaction

PMID:11058758

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. abnormal acute phase protein level (MP:0010211)
b. abnormal circulating orosomucoid 1 (new term)
c.  abnormal circulating orosomucoid 1 (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:mlkaldunski

**Comment by anna-anagnostop** [comment]
abnormal circulating orosomucoid 1 level
DEF: deviation from the normal levels of orosomucoid 1 (ORM1, also known as alpha1-acid glycoprotein 1), an acute phase plasma protein known to increase during inflammation; it has many activities including, but not limited to, acting as an acute-phase reactant and disease marker, modulating immunity, binding and carrying drugs, maintaining the barrier function of capillary, and mediating the sphingolipid metabolism
[PMID: 31930291](https://pubmed.ncbi.nlm.nih.gov/31930291/), [PMID: 25711902](https://pubmed.ncbi.nlm.nih.gov/25711902/)

see also P[MID: 31026065](https://pubmed.ncbi.nlm.nih.gov/31026065/): Orosomucoid (ORM) is an acute-phase reaction protein. It is mainly produced by the liver, and also expressed in extrahepatic organs, such as the lung, spleen, and lymph (Luo, Lei, Sun, Liu, & Su, 2015). ORM is a heavily glycosylated (45%) protein with a low pH of 2.8–3.8 and a molecular weight ranging from 37 to 54 kDa. There are two subtypes of ORM in humans (ORM1 and ORM2), three subtypes in mice (ORM1, ORM2, and ORM3), and only one type in rats. In humans and mice, ORM1 accounts for more than 75% of total serum ORM and is the main elevated subtype during acute-phase response. ORM has multiple biological activities, such as transporting drugs, regulating immunity, maintaining capillary barrier, regulating lipid metabolism, and so forth (Luo et al., 2015). The increase of serum ORM concentration often reflects the body's pathological state, such as trauma, infection or shock.

**Comment by sbello** [comment]
MP:0021037 - abnormal
MP:0021038 - increased
MP:0021039 - decreased
